.github/workflows/tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         os: ['ubuntu-latest']
         python-version: [
           '3.9',
-          '3.13',
+          '3.14',
```

Comment:
The latest `duckdb` GA wheel package is not available for Python 3.14 yet, so it needs to be compiled, which increases the build time on CI significantly from 1 minute to 18 minutes.

- https://pypi.org/project/duckdb/1.4.1/#files